youtube_dl/extractor/polskieradio.py

```diff
@@ -96,4 +151,6 @@ def _real_extract(self, url):
         title = self._og_search_title(webpage).strip()
         description = strip_or_none(self._og_search_description(webpage))
 
-        return self.playlist_result(entries, playlist_id, title, description)
+        result = self.playlist_result(entries, playlist_id, title, description)
+        result['_type'] = 'multi_video'
```

Comment:
The `multi_video` type seemed to me to be more appopriate for describing a list of media files that belong to a single article/broadcast, as noted in `common.py`. The `playlist` type seems to better fit the case of multiple broadcasts of a particular programme.